src/lib/bcmp/bcmp.cpp

```diff
@@ -241,7 +241,7 @@ static bool bcmp_dfu_tx(bcmp_message_type_t type, uint8_t *buff, uint16_t len) {
   \param *netif lwip network interface to use
   \return none
 */
-BmErr bcmp_init(NvmPartition *dfu_partition, Configuration *user_cfg, Configuration *sys_cfg,
+BmErr bcmp_init(NvmPartition *dfu_partition, Configuration *sys_cfg,
```

Comment:
remove user_cfg since we aren't passing it into bcmp_config_init anymore. (sys_cfg is used by dfu still, so it will be removed next).